src/ui/camera.js

```diff
@@ -194,7 +238,7 @@ class Camera extends Evented {
      * @fires zoomend
      * @returns {Map} `this`
      * @example
-     * // zoom the map to 5
+     * // zoom the map to 5 without an animated transition
```

Comment:
```suggestion
     * // Zoom to the 5th zoom level without an animated transition
```

I think we should always associate the zoom level number (5) with the words "zoom level." Otherwise it's not entirely clear what unit the value is in.